share/src/main/java/io/zensoft/share/service/diesel/RequestSender.java

```diff
@@ -0,0 +1,7 @@
+package io.zensoft.share.service.diesel;
+
+import io.zensoft.share.model.diesel.RequestResponse;
+
+public interface RequestSender {
+    RequestResponse getFilledResponseFromSender(RequestResponse requestResponse);
```

Comment:
Request and Response is two different notions in network programming. So i think its not a good idea to use this words together

Reply:
There I save Responses from POST/GET requests. And it is logic to name it RequestResponse.

Yes, I understand your vision. What name of class do you recommend?

Reply:
Response

Reply:
I think, it is not so understandable, when we name it just Response. Response from what? Which Response?